packages/grafana-ui/src/components/Switch/Switch.tsx

```diff
@@ -14,16 +14,15 @@ export interface Props {
 export interface State {
   id: any;
 }
-
-export class Switch extends PureComponent<Props, State> {
+ export class Switch extends PureComponent<Props, State> {
```

Comment:
some formatting issue here? Aren't we running prettier precommit? 🤔

Reply:
yea, strange